deployment/ansible/roles/driver.app/tasks/firewall.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Enable firewall
+  ufw: state=enabled policy=reject
+
+- name: Allow ssh access
```

Comment:
I was going to say this could be written more succinctly using `with_items` but then I realized that we'd lose the output of each port that gets opened, which I kind of like. But you might have different preferences, I think either way would be fine.